core/network/src/sync.rs

```diff
@@ -54,9 +54,15 @@ struct PeerSync<B: BlockT> {
 	pub recently_announced: VecDeque<B::Hash>,
 }
 
+#[derive(Copy, Clone, Eq, PartialEq, Debug)]
+enum AncestorSearchState<B: BlockT> {
+	LinearSearch(NumberFor<B>),
```

Comment:
Is this just a number of steps done in this mode? A comment would be nice. It can also be a regular `u32`

Reply:
Yes, commented and changed to `u32`.